iterator/git.go

```diff
@@ -239,7 +248,11 @@ func (obj *Git) Recurse(ctx context.Context, scan interfaces.ScanFunc) ([]interf
 
 	// make a unique ID for the directory
 	// XXX: we can consider different algorithms or methods here later...
-	sum := sha256.Sum256([]byte(obj.URL))
+	// Added commit hash due to error when scanning
+	// https://webrtc.googlesource.com/src using different commit hashes and
+	// without different commit hashes.
```

Comment:
Add a little comment about what we're doing here and uniqueness please.

Reply:
Fixed